File: 14-LongestCommonPrefix/14-LongestCommonPrefix.py
# Last updated: 8/24/2026, 3:48:01 PM
1class Solution:
2    def longestCommonPrefix(self, strs):
3        strs.sort()
4        s = ""
5        i = 0
6        length = len(strs)
7
8        while i < len(strs[0]):
9            if strs[0][i] == strs[length - 1][i]:
10                s += strs[0][i]
11            else:
12                break
13            i += 1
14
15        return s

File: 14-LongestCommonPrefix/14-LongestCommonPrefix.java
// Last updated: 8/24/2026, 3:53:28 PM
1class Solution {
2    public int lengthOfLastWord(String s)
3     {
4        s = s.trim();
5        int last_count =0;
6       
7        for(int i=s.length()-1;i>=0;i--){
8        if(s.charAt(i)==' '){
9            break;
10        }
11            last_count++;
12        
13        }
14    
15    return last_count;
16     }
17}